Provide registry credentials during  SAIF installation (#90)
SUSE/doc-suse-ai: tasks/aif-credentials-helm.adoc

include::../common/generic-attributes.adoc[]
// overridden title?
ifdef::override-title[]
= {override-title}
:revdate: 2026-07-31
:page-revdate: {revdate}
endif::[]
ifndef::override-title[]
= Providing registry credentials during {productname} installation
endif::[]
// overridden abstract?
ifdef::override-abstract[]
{override-abstract}
endif::[]
ifndef::override-abstract[]
ifeval::["{PROF_PRODUCT}" == "saif"]
To access applications and blueprints from {sappco} and {sregistry}, you need to provide valid credentials.
endif::[]
ifeval::["{PROF_PRODUCT}" == "saifnv"]

> To access applications and blueprints from {sappco}, {sregistry} and {nvidia} catalog, you need to provide valid credentials.

Reviewer: ```suggestion
To access applications and blueprints from the {sappco}, {sregistry} and {nvidia} catalogs, you need to provide valid credentials.
```

endif::[]
By default, you can configure the credentials after the {productname} installation is complete as described in xref:ai-factory-post-deployment[].

However, you can specify the credentials during the {productname} installation process as well.
endif::[]

// erase the flag for future overrides
:override-abstract!:
:override-title!:

[#aif-helm-credentials-procedure]
.Providing registry credentials during {helm} CLI installation

. Download and display the default values for the {productname} installation.
+
[source,bash,subs="+attributes"]
----
{prompt_user}helm show values oci://ghcr.io/suse/chart/aif-operator --version {aif-latest-version}
----

> . Create a `aif-values.yaml` file and copy the part that deals with registry credentials, for example:

Reviewer: ```suggestion
. Create an `aif-values.yaml` file and copy the part that deals with registry credentials, for example:
```

+
[source,yaml]
----
credentials:
  # credential for oci://dp.apps.rancher.io/charts
  applicationCollection:
    username: ""
    password: ""
  # credential to pull NGC artifacts
  # e.g. https://helm.ngc.nvidia.com/nvidia
  nvidia:
    username: ""
    password: ""
  # credential for oci://registry.suse.com/ai/charts
  suseRegistry:
    username: ""
    password: ""
----

> . Provide the credentials to individual registries on the related `username:` and `password:` fields.

Reviewer: ```suggestion
. Provide the credentials to individual registries in the related `username:` and `password:` fields.
```
